db/dataset.py

```diff
@@ -115,7 +115,7 @@ def update(dataset_id, dictionary, author_id):
             dictionary["description"] = None
 
         connection.execute("""UPDATE dataset
-                          SET (name, description, public, author) = (%s, %s, %s, %s)
+                          SET (name, description, public, author, last_edited) = (%s, %s, %s, %s,current_timestamp)
```

Comment:
watch the space after the comma here too. I tend to use `now()` instead of `current_timestamp`. they both mean the same thing